tests/components/bang_olufsen/test_media_player.py

```diff
@@ -167,26 +168,26 @@ async def test_async_update_playback_metadata(
 
 
 async def test_async_update_playback_error(
-    hass: HomeAssistant, mock_mozart_client, mock_config_entry
+    hass: HomeAssistant,
+    caplog: pytest.LogCaptureFixture,
+    mock_mozart_client,
+    mock_config_entry,
 ) -> None:
     """Test _async_update_playback_error."""
 
     mock_config_entry.add_to_hass(hass)
     await hass.config_entries.async_setup(mock_config_entry.entry_id)
 
     # The async_dispatcher_send function seems to swallow exceptions, making pytest.raises unusable
-    with patch("homeassistant.helpers.dispatcher._LOGGER.error") as mock_logger:
-        async_dispatcher_send(
-            hass,
-            f"{TEST_SERIAL_NUMBER}_{WebsocketNotification.PLAYBACK_ERROR}",
-            TEST_PLAYBACK_ERROR,
-        )
+    async_dispatcher_send(
```

Comment:
Side note: We should not use the dispatcher during tests. It's an integration detail.

Reply:
Yes, this has been rectified in another PR. https://github.com/home-assistant/core/blob/156e39ebb23e2ed84a8b05d45e642280e3f7ac14/tests/components/bang_olufsen/test_media_player.py#L213-L218

The comment should probably be updated.

Reply:
Very good! Thanks!